content/blog/meeting-format-changes.md

```diff
@@ -0,0 +1,30 @@
+---
+date: "2017-07-05T00:12:00-04:00"
+draft: false
+title: "More Info on the Main Meeting Improvements"
+---
+
+We have a lot of tweaks/improvements planed for the main meeting format and to help set expectations I wanted to post some notes beforehand.
+
+First, the meeting (Thursday July 13th) will be held at IndyHall instead of the Apple Store. We announced this back in May and we hope the more open floor plan of IndyHall will let us do things we could not before, as well as host more people.
+
+Second, the meeting will be returning to it's previous times of 6:30pm to 9:00pm. We hope the later starting time will help people who have to commute to CocoaHeads from a job somewhere far away. If you do arrive early there is plenty of desk space to crash on while we setup. There are also numerous coffee shops in the area if that is more your cup of tea, er coffee.
+
+Third, we are expanding the social time and introducing a "problem board" system to help people who are struggling with a code issue or looking for help in some other way. This kind of pairing up is something that has been very successful at Side Project Saturdays and we want to bring a piece of it to our main meeting.
+
+When you arrive you'll be able to post your "problem/question" to a board which will be on the projector. We'll do our best to match you up to someone who can help or let that someone come to you during social time. Some examples of problems or questions include:
+
+* Bugs and other Broken Behavior
+* Code Review
+* App Idea Review
+* UI Review
+* App Architecture Review / Brainstorms
+* Framework Introductions
+* Build Automation
+* Testing Improvements
+* Resume Review
+* Job Interview Practice
+
+So please, take some time before the meeting to draft up something you'd like some help with and we'll do our best to help.
```

Comment:
So please feel comfortable sharing an issue and making use of the different expertise of your peers. You'll do the same for someone else down the line.